### DISCUSSION TITLE
Welcome to user_login_and_register Discussions!

### DISCUSSION THREAD

**Discussion by itsvinayak** [itsvinayak/user_login_and_register#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hey, Vinayak. I was wondering, where is the database created to store the data of the registered users? how do i find that database?

**Reply by ghost** [reply]
Found it. Run the server and then open Django administration (URL/admin). The user data should be there
